woocommerce/woocommerce-ios
# [Better customer selection] Add username to model

Part of: #10392 


## Description

With this PR we add a new model containing the username property for customers. This is the first step to showing that username in the customer selector lists. We add it in both `WCAnalyticsCustomer` (only `Networking/Yosemite`) and `Customer` (`Storage` and `Networking`)

## Testing instructions

Ensure that unit tests pass correctly. You can also make sure that the model migrates correctly by running a version with model 92 (trunk) and then installing this version over.

## Screenshots

N/A


---
